src/principle_comp.c

```diff
@@ -0,0 +1,61 @@
+/******************************************************
+* Author : Amuleen Gulati 
+* Function : principle component 
+* Application : This function computes the principle component values
+* for the input eigen values and eigen vectors
+* Input : number of sensors n as integer, eigen vector matrix T[n][n], eigen value vector D[n]   
+* Output : vector of size n of principle components
+*/
+
+#include "..\include\principle_comp.h"
+#include "..\include\transpose.h"
+#include "..\include\multiply.h"
+
+double* principle_component(int n, double** D, double* T){
+
+/* Create a vector to store principal component values */
+double* y;
```

Comment:
Bad variable name.

Reply:
I have created these variable names as they are in the given algorithm. So its easier to identify without any loss of generality.